### ISSUE TITLE
[Logging] Console Spammed With NullPointerException's

### ISSUE THREAD

**Issue by xlxAciDxlx** [Lycanite/LycanitesMobs#196]
Looking at the source I'm assuming a mob is trying to spawn but errors out. The console is spammed repeatedly with `NullPointerException` errors every few seconds.

[Gist](https://gist.github.com/xlxAciDxlx/22f5249387264d99607dceecb6a5333a) | [Erroring Lines](https://github.com/Lycanite/LycanitesMobs/blob/master/java/lycanite/lycanitesmobs/core/spawning/SpawnTypeBase.java#L512-L516)

**Comment by Lycanite** [comment]
Hmm that's very strange, do you get anything else or just that exception, the line "Unable to instantiate an entity from SpawnInfo: ..." should try to log after which should give more information about it.

**Comment by xlxAciDxlx** [comment]
Should it log it into the console as well? I think I've seen it do a full stack trace before, but I'll have to search through my log files to find it.

**Comment by Lycanite** [comment]
That could help, also there are some debug settings in I think either in lycanitesmobs-debug.cfg if there might be a debug section in the general config instead, if you turn on the mob spawning debug information it should spit out a lot more information on which mob is spawning and what spawner is spawning it, etc.

**Comment by xlxAciDxlx** [comment]
Unless I'm missing something in my logs, I'm not seeing anything helpful other than the spammed message(s). I'll turn those settings on and get back to you (it's currently almost 5AM for me lol)

**Comment by gigafunk** [comment]
[stacktrace.txt](https://github.com/Lycanite/LycanitesMobs/files/1099438/stacktrace.txt)

Is this the stacktrace? 

I have a 1.10.2 forge 3216 ....sponge 1.10.2-2281-5.2.0-BETA-2401--which is built against forge 2281-one of the tests I want to do is backing my forge up to 2281...but I am just starting to research and figure this out and saw this similar error and was hoping this would be a clue. I can get full clean logs or whatever will help, that log is 50 megs of that, and i would want to turn on debug messages.  This error showed up when a player want to a new area of the map. it was pregenned, but this is the first time a player went there.  at first there is a message

  [11:28:37] [Server thread/INFO] [STDERR/]: [lycanite.lycanitesmobs.LycanitesMobs:printWarning:412]: [LycanitesMobs] [WARNING] Null coordinates! This spawn type might never be able to find coordinates as it has no materials or blocks set, not even air.

Then the stacktraces start and repeat.

worth mentioning this server has 190 mods on it, including sponge, and I cant believe its all works so well. My last server was 1.5 so we have come a long way lol.    I am going to spentd a few hours testing things in a bit, no sponge and new worlds and blah blah, but figured I would start here in case you can shed any light on which direction to test.

I have doomlike dungeon. could this be that mod spawning these mobs and delivering these messages? or would that show up as doomlike errors? 

Thanks for the cool mobs!   Any info that could help me would be appreciated,  like do you think this could be a problem with one type of mob I may be able to disable?    Thanks again!

**Comment by Lycanite** [comment]
That is indeed a stacktrace, it looks like a mob is trying to spawn but it's class or something cannot be found, I can look into suppressing this, having a lot of mods can make it hard to determine the problem but I'll see what I can find!

**Comment by gigafunk** [comment]
It maybe started when we updated cofh and thermal stuff and they introduced the oil retrogen.  we are not sure if we just visited an area where this mob was trying to spawn and that started it or the oil retrogen. I think their new oil is a liquid in the world, and i thought maybe it was interacting with the ooze class crreature trying to spawn.  Your mobs are so nice man. I am a little worried about the reported ai on 1.10 heaviness as we where having some trouble, particulartly with the shade i think it was called? we had to disable that one completely.   Its a bare metal debian server , so its got plenty of cores and ram to itself, and our bigger 1.7.10 pack runs great on it. (not at the same time we only run one) We had to pull it off for now, the logs were HUGE and had to keep clearing them. But i do like your mobs, they are the best mobs out there. Thanks for your work!
